src/main/java/com/gooddata/md/Updatable.java

```diff
@@ -0,0 +1,7 @@
+package com.gooddata.md;
+
+/**
+ * Marker interface for metadata objects which can be updated using PUT on objects URI (see MetadataService.updateObj).
```

Comment:
"using PUT on objects URI" is a bit misleading. Either speak with whole sentence - eg. "using HTTP PUT method..." or remove.
ad java doc link to `updateObj()`